modules/zellij/hm.nix

```diff
@@ -1,26 +1,29 @@
 { config, lib, ... }:
 
 {
-  options.stylix.targets.zellij.enable =
-    config.lib.stylix.mkEnableTarget "zellij" true;
+  options.stylix.targets.zellij.enable = config.lib.stylix.mkEnableTarget "zellij" true;
 
-  config = lib.mkIf (config.stylix.enable && config.stylix.targets.zellij.enable) {
-    programs.zellij.settings = {
-        theme = "stylix";
-        themes.stylix = with config.lib.stylix.colors.withHashtag; {
-            bg = base03;
-            fg = base05;
-            red = base08;
-            green = base0B;
-            blue = base0D;
-            yellow = base0A;
-            magenta = base0E;
-            orange = base09;
-            cyan = base0C;
-            black = base00;
-            white = base07;
-        };
-    };
-  };
+  config =
+    lib.mkIf
+      (config.stylix.enable && config.stylix.targets.zellij.enable && config.programs.zellij.enable)
+      {
+        xdg.configFile."zellij/themes/stylix.kdl".text = with config.lib.stylix.colors.withHashtag; ''
```

Comment:
Do we not have to declare the [Zellij theme](https://zellij.dev/documentation/options.html?highlight=theme#theme):

```nix
let
  theme = "stylix";
in
  inherit (programs.zellij.settings) theme;
  xdg.configFile."zellij/themes/${theme}.kdl".text = /* ... */
```

Reply:
> Technically no, because the style is called default, if we named the font anything else (a.k.a. stylix) then yes

Is the current style not called `stylix.kdl`, e.g. not `default`?

Reply:
That is the file not the name of the theme

Reply:
See here: https://github.com/danth/stylix/pull/616/files#r1841758001